src/components/Popover/Popover.driver.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { POPOVER_DATA_KEYS } from './dataHooks';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+
+export interface PopoverDriver extends BaseUniDriver {
+  withArrowTop(): any;
+  withTitle(): any;
+  isRightArrow(): any;
+  withShadow(): any;
+  withArrow(): any;
+}
+
+export const popoverDriverFactory = (base: UniDriver): PopoverDriver => {
+  return {
+    ...baseUniDriverFactory(base),
+    async withArrow() {
+      return (await base.attr(POPOVER_DATA_KEYS.WithArrow)) === 'true';
+    },
+    async withShadow() {
+      return (await base.attr(POPOVER_DATA_KEYS.WithShadow)) === 'true';
+    },
+    async isRightArrow() {
+      return (await base.attr(POPOVER_DATA_KEYS.RightArrow)) === 'true';
+    },
+    async withTitle() {
+      return (await base.attr(POPOVER_DATA_KEYS.Title)) === 'title';
+    },
+    async withArrowTop() {
+      return (await base.attr(POPOVER_DATA_KEYS.ArrowTop)) === '50px';
```

Comment:
why is this hardcoded to `50px`?
This should be something like:
```suggestion
    async hasArrowTop(value: string | number) {
      return (await base.attr(POPOVER_DATA_KEYS.ArrowTop)) === value;
```

or maybe just:
```suggestion
    async getArrowTop() {
      return base.attr(POPOVER_DATA_KEYS.ArrowTop);
```

Reply:
We might not have talked about it, 
but the drivers are also exported for the consumers to be used as testkits.
So the API of the driver is also a contract we have with our consumers